### ISSUE TITLE
Wiimotes not appearing in SteamVR

### ISSUE THREAD

**Issue by 2haloes** [ajayyy/Wiimote-FreePIE-VR-Controls#9]
Hello, 

I've set this up how the readme described (installing the programs and replacing files) and FreePIE reports activity from the controllers but SteamVR will not recognize them. 

No errors are reported and I've double checked everything (the fake suxense files are in the right place).

I'm using Windows 10 with the Dolphinbar to connect the deivces to

**Comment by ajayyy** [comment]
Did you make sure to rename sizsense fake? Do you have the hydra driver installed

**Comment by 2haloes** [comment]
Yes and yes, I copied and renamed the file fake twice (for 64 and 32 bit) and I kept the originals as BAK files

**Comment by ajayyy** [comment]
Hmm... I'm not sure what the issue would be then. Maybe you can test if it is at least emulating hydras by running the hydra testing program.

**Comment by 2haloes** [comment]
I tried running HydraRyder and the controllers were seen and the controlled worked, however the motion controls didn't (so it was just aimed in the middle the whole time. I checked the leap control panel and I noted that it doesn't see the devices, is that normal?

**Comment by ajayyy** [comment]
The leap control panel should be green on the status bar showing that the leap motion is plugged in.

**Comment by arg-quarky** [comment]
do i need a leap motion device?
